### PR TITLE
Create redirect-request.md

### PR DESCRIPTION
**Pull request by mnorthuis**
Template for requesting redirects

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/redirect-request.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+assignees: mnorthuis
+
+---
+
+### Instructions
+- Requests for redirects need to be submitted **AT LEAST** 2 weeks in advance.  Some redirects take significant amount of time and coordination to implement, so start the process as soon as you know you will need one. 
+- Please add your team's label to this issue and complete the information below.
+- This issue will be used from initial request through implementation of the redirect to ensure all individuals working on this are notified of status updates.  Please do not create multiple tickets to track differnt steps.
+- It is the responsibility of the requesting team to notify VA stakeholders of pending redirect.  
+
```
Comment (meganhkelley) [inline .github/ISSUE_TEMPLATE/redirect-request.md]: We should minimize the "Instructions" area, assuming few people will read it completely through. These all do seem important, but maybe we can cut excess words to make it more easily skim-able. Proposed changes: 

- Bullet 1, cut the second sentence? Not sure?
- Cut bullet 2 (and just add to steps below)
- Bullet 3, use "issue" or "ticket" consistently. Typo in "different"
- Bullet 4 specify "you" — "It is YOUR responsibility, as the team requesting the redirect, to notify VA and other stakeholders of pending redirect."

File: .github/ISSUE_TEMPLATE/redirect-request.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+### Redirects Needed
+
+Current URL  |  Redirect to new URL
+---  |  ---
+*enter current URL* | *enter new URL*
+
+
+### Steps to complete
+- [ ] Above information is provided and issue is tagged and assigned appropriately - *@ requesting team*
```
Comment (meganhkelley) [inline .github/ISSUE_TEMPLATE/redirect-request.md]: "Above information is provided, issue is tagged with your team label, and assigned to @mnorthuis"

File: .github/ISSUE_TEMPLATE/redirect-request.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+- [ ] All appropriate VA stakeholders are notified of pending redirect - *@ requesting team*
```
Comment (meganhkelley) [inline .github/ISSUE_TEMPLATE/redirect-request.md]: "VA and other"? Should we be more specific? Less specific?

File: .github/ISSUE_TEMPLATE/redirect-request.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+- [ ] Redirect request is vetted and documented and implementation plan is clear - *@ Content & IA team*
+- [ ] Request is assigned to appropriate team for implementation - *@ Content & IA team*
+- [ ] Implementation team completes work - *@ Implementation team*
+- [ ] Implementation pushed live and redirect is validated in production - *@ all*
+- [ ] Ticket is closed - *@ requestion team*
```
Comment (meganhkelley) [inline .github/ISSUE_TEMPLATE/redirect-request.md]: "Requesting" (typo)

**Review by meganhkelley — APPROVED** [review]
Overall this looks good to me! I'm inclined to merge to master and then check it out in ZH to see what we're forgetting/see what it looks like. And maybe we can clean out some unused templates while we're at it to make that list in ZH less crowded. 

Took a look at Public Websites' workflow and I think we're incorporating all the info they need! The only missing piece that I noticed is "Requestor POC" (in Jen's doc) — our hope is that this will be the person creating the ticket. But, should we add it into the template as "Redirect requested by (if other than yourself)" or something? Alternately we could mention it in the italics of the "Description" section, I just worry people will ignore those italics.

File: .github/ISSUE_TEMPLATE/redirect-request.md (hunk 3 above)
Comment (mnorthuis) [reply .github/ISSUE_TEMPLATE/redirect-request.md]: VA contacts are the most critical here.

**Pull request merged**